examples/multigpu_advtrain/run.py

```diff
@@ -0,0 +1,317 @@
+"""
+This example adversarially trains a model using iterative attacks.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import six
+import math
+import time
+import os
+import logging
+import random
+
+import tensorflow as tf
+from tensorflow.python.platform import app
+from tensorflow.python.platform import flags
+
+from cleverhans.utils_mnist import data_mnist
+from cleverhans.utils_tf import model_loss
+from cleverhans.utils_tf import batch_indices
+from cleverhans.utils_tf import _FlagsWrapper
+import cifar_input
+
+from make_model import make_model
+from evaluator import Evaluator, get_attack
+
+
+FLAGS = flags.FLAGS
+_data_path = {'svhn':  '/ssd1/datasets/svhn/',
+              'cifar10':  '/ssd1/datasets/cifar-10/'}
+if "CIFAR10_PARDIR" in os.environ:
+    _data_path['cifar10'] = os.environ['CIFAR10_PARDIR']
+
+
+def preprocess_batch(images_batch, preproc_func=None):
```

Comment:
There's another function with the same name in manager.py. How are they similar / different?

Reply:
Fixed in 9078122.